src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -696,22 +696,25 @@ export class BaseEditCourseForm extends React.Component {
             courseSubmitting={submitting}
             {...this.props}
           />
+          <Link to={`/courses/${uuid}/rerun`}>
+            <button
+              id="new-run-button"
```

Comment:
Nit: I might check with Alasdair, but I think using ids for styling is typically frowned upon and it's better to use a class.